spring-boot-modules/spring-boot-mvc-5/src/main/java/com/baeldung/problemdetails/web/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,22 @@
+package com.baeldung.problemdetails.web;
+
+import com.baeldung.problemdetails.model.InvalidInputException;
+import java.net.URI;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ProblemDetail;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
```

Comment:
An example that the formatter is not being applied is the order of the imports and the separation between the different groups:
Java

Jakarta

javax

organization

com

Reply:
Not sure why in Intellij code formatter is not organizing the imports. I used Eclipse to format the imports for now.